### ISSUE TITLE
Naming Convention Analysis

### ISSUE THREAD

**Issue by wellso** [prahladyeri/VisualAlchemist#27]
Naming convention inconstancy is a major headache in many projects I inherit; I have been thinking of writing a feature to analyse an schema to see if it fits a pre-defined naming conventions (e.g. singular vs plural table names, CamelCase vs snake_case etc.). Would you consider this useful as a feature?

**Comment by EmilMoe** [comment]
Even though I'm working alone, yes. Sometimes you forget your own best practices.

**Comment by prahladyeri** [comment]
Hey @wellso Welcome to Visual Alchemist!

I was pretty new to JavaScript when I started coding for VA. However, in the true spirit of JavaScript, I've followed camelCasing wherever possible. For example, `stringTypes`, `readCookie` and `loadCanvasState`. Let me know what you felt odd in the code?

I'm feeling right now that the bigger problem here is architecture. I've put everything in this jQuery based `app.js` god script, so its a huge pain to make any changes now! I'm thinking of using `BackBone.js` to create a modular structure in VA.
